src/main/java/kitchenpos/products/infra/ProductPriceChangesEvent.java

```diff
@@ -0,0 +1,25 @@
+package kitchenpos.products.infra;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+import org.springframework.context.ApplicationEvent;
+
+public class ProductPriceChangesEvent extends ApplicationEvent {
```

Comment:
이벤트는 현재 기준으로 과거에 벌어진 것을 표현하기 때문에 이벤트 클래스의 이름도 일반적으로 과거시제를 사용하는 것을 권장합니다.

```suggestion
public class ProductPriceChangedEvent extends ApplicationEvent {
```

Reply:
@HackSung 

음.. 제가 이해한건 
ProductPriceChangedEvent 클래스는 외부로 발행하는 영역이기에 infra 영역으로 가는게 자연스럽고
MenuEventListener 클래스는 외부로부터 응답을 받고 이벤트와 도메인 모델 간의 중재자 역할이므로 application 영역으로 가야한다

라고 이해했습니다!